flake.lock

```diff
@@ -268,16 +268,16 @@
     },
     "nixpkgs": {
       "locked": {
-        "lastModified": 1669546925,
-        "narHash": "sha256-Gvtk9agz88tBgqmCdHl5U7gYttTkiuEd8/Rq1Im0pTg=",
+        "lastModified": 1676817468,
+        "narHash": "sha256-ovuJ1jQOC2/EEibufBkXmSN/O9mLx80Wh7aDmHmHAhA=",
         "owner": "NixOS",
         "repo": "nixpkgs",
-        "rev": "fecf05d4861f3985e8dee73f08bc82668ef75125",
+        "rev": "0cf4274b5d06325bd16dbf879a30981bc283e58a",
```

Comment:
How does it work,  are we using different packages when running in cicero rather than with regular nix?  So this change will impact only cicero runs?

Reply:
No, we could do that but we don't. This line says we want to use the same nixpkgs for cicero:
https://github.com/input-output-hk/cardano-ledger/blob/d86964db9112031f1266363385babb0ff99216b3/flake.nix#L16

If we removed that line it would fall back to the nixpkgs version that happens to be defined in tullia's flake. That would be fine but unless there is a need to use different nixpkgs versions for different things, it's generally a good idea to only use one nixpkgs and share that with everything in a flake. Every nixpkgs instance needs about 300 MB of memory to evaluate (IIRC). So by sharing one we can speed up evaluation. Also wouldn't it be a good idea to update from 22.05 to 22.11 for our package build as well?

Reply:
I still don't understand, sorry! 
So: we ARE using the same nixpkgs for cicero because of the  ` inputs.nixpkgs.follows = "nixpkgs"; ` . 
But we are not using flakes for anything else, at least not as a developer/CI. 

So then what is the effect of updating nixpkgs here,  if the package build is using the nixpkgs from sources.json (by not using flakes), and  we have this setting: ` inputs.nixpkgs.follows = "nixpkgs"; `  ?
(Sorry if I'm missing something very obvious)

Reply:
Oh, it was me who missed something very obvious! :smile: I didn't look properly and just assumed a normal flake-based approach in this repo but we're using niv. So yes, this change will only impact cicero runs.

Reply:
Thanks for the explanation :)